src/server/webkit/wkPage.ts

```diff
@@ -591,6 +591,13 @@ export class WKPage implements PageDelegate {
       }
       promises.push(session.send('Page.setForcedAppearance', { appearance }));
```

Comment:
Let's fix the bug here as well - we have to send this command even when colorScheme changes from something to null.